# This is a BitKeeper generated diff -Nru style patch.
#
# ChangeSet
#   2004/07/03 19:29:45-03:00 marcelo@logos.cnet 
#     Add missing bracket to inode_change_ok() fix
#   TAG: v2.4.27-rc3
# 
# fs/attr.c
#   2004/07/03 19:28:29-03:00 marcelo@logos.cnet +1 -1
#   Add missing bracket to inode_change_ok() fix
# 
--- fs/attr.c	2004-07-09 17:30:54.000000000 -0400
+++ fs/attr.c	2004-07-09 16:17:05.000000000 -0400
@@ -33,7 +33,7 @@
 
 	/* Make sure caller can chgrp. */
 	if ((ia_valid & ATTR_GID) &&
-	    (!in_group_p(attr->ia_gid) && attr->ia_gid != inode->i_gid) &&
+	    (!in_group_p(attr->ia_gid) && attr->ia_gid != inode->i_gid)) &&
 	    !capable(CAP_CHOWN))
 		goto error;
 
